UPGRADING.md

```diff
@@ -18,6 +18,22 @@ limitations under the License.
 Upgrading
 =========
 
+# From 1.7 to 1.8
+
+Upgrades from 1.7 to 1.8 are be possible with little effort as no changes were made at the data layer and RPC changes
```

Comment:
haha, I copied from the previous one.  It is there too, I'll clean them both up.